.github/workflows/release.yml

```diff
@@ -72,22 +78,6 @@ jobs:
           asset_name: marina-v3-${{ steps.next_version.outputs.version }}.zip
```

Comment:
here you uplaoding only the v3?

Reply:
Yep, asset_path is: `web-ext-artifacts/v3/......zip`

Reply:
we need to upload both v2 and v3 no?

Reply:
nvm I see the extra step later